README.md

```diff
@@ -31,17 +31,36 @@ pip install git+https://github.com/bokulich-lab/RESCRIPt.git
 ```
 
 ### Option 2: Install within QIIME 2 environment
```

Comment:
say "before version 2023.9"?

Reply:
I assume the install instructions for `2024.2` are going to be largely similar to `2023.2` - `2023.7` right? That is pointing to a `.../202X.Y/tested/` folder? If so, I think labelling as 'before' or 'after' `2023.9` might be confusing, as that is the only odd ball. Which is why I set it up as a *note for `2023.9`*. 🤷 

Also, this makes me think... should we add instructions for older versions, `2022.8` and earlier? Or would that just make things more confusing? I figure no, as folks can ask for help with older versions.

Reply:
Oh wait I forgot, pre- `2023.9` the tested folders will be `https://packages.qiime2.org/qiime2/2023.7/tested/` whereas `2024.2` will be: `https://packages.qiime2.org/qiime2/2024.2/amplicon/released/`. So, the pre- post labeling will work. :-)